docs/installation/source.rst

```diff
@@ -37,36 +37,33 @@ please follow the directions :ref:`here <contributing>`.
        On Fedora Linux, you must replace ``pip`` with ``pip-python`` in the
        following steps.
 
-#. Then you'll need to install GStreamer 0.10 (>= 0.10.31, < 0.11), with Python
-   bindings. GStreamer is packaged for most popular Linux distributions. Search
-   for GStreamer in your package manager, and make sure to install the Python
+#. Then you'll need to install GStreamer 1.x (>= 1.2), with Python bindings.
+   GStreamer is packaged for most popular Linux distributions. Search for
+   GStreamer in your package manager, and make sure to install the Python
    bindings, and the "good" and "ugly" plugin sets.
 
    If you use Debian/Ubuntu you can install GStreamer like this::
 
-       sudo apt-get install python-gst0.10 gstreamer0.10-plugins-good \
-           gstreamer0.10-plugins-ugly gstreamer0.10-tools
+       sudo apt-get install python-gst-1.0 gstreamer1.0-plugins-good \
+           gstreamer1.0-plugins-ugly gstreamer1.0-tools
 
    If you use Arch Linux, install the following packages from the official
    repository::
 
-       sudo pacman -S gstreamer0.10-python gstreamer0.10-good-plugins \
-           gstreamer0.10-ugly-plugins
+       sudo pacman -S gst-python gst-plugins-good gst-plugins-ugly
```

Comment:
The basic Gst API probably works without the overrides provided by gst-python, but I don't know to what degree we rely on the overrides (ref. http://cgit.freedesktop.org/gstreamer/gst-python/tree/gi/overrides/Gst.py). The gst-python source seems Python 2+3 compatible, so there really should be a gst-python package for Python 2 in Arch. For now, I think the workaround is to add python2-gobject to the install command.